extruct/opengraph.py

```diff
@@ -32,6 +32,8 @@ def extract_items(self, document, base_url=None):
             for el in head.xpath('meta[@property and @content]'):
                 prop = el.attrib['property']
                 val = el.attrib['content']
+                if prop == '' or val == '':
```

Comment:
Should `prop` or `val` be stripped to also consider multiple spaces/non-visible-characters as empty?

Reply:
Yes, I actually did that initially and then I removed the .strip() because I thought maybe we don't want to change the actual values?
Writing the code is easy, not sure what are the implications to the other libraries using this..

Reply:
But again, HTML should ignore extra spaces in general... So...
I'll add the .strip()

Reply:
Ok!